.github/workflows/push-on-develop.yml

```
@@ -35,19 +44,71 @@
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY_IMAGE }}
+
+      - name: Log in to Docker Hub 🔐
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+
+      - name: Create and push manifest list 📦
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:2:3: Quote this to prevent word splitting

(shellcheck)